packages/devtools_test/.gitignore

```diff
@@ -29,3 +29,4 @@ migrate_working_dir/
 .dart_tool/
 .packages
 build/
+devtools_options.yaml
```

Comment:
Is this intended?

Reply:
Yeah the devtools_options.yaml file keeps getting added for devtools_test, and we do not need devtools extensions for that package.